packit_service/worker/helpers/build/koji_build.py

```diff
@@ -67,20 +71,28 @@ def __init__(
     def is_scratch(self) -> bool:
         return self.job_build and self.job_build.scratch
 
+    @staticmethod
+    def configured_or_default(targets: Set[str]) -> Set[str]:
+        return targets or {DEFAULT_VERSION}
```

Comment:
```suggestion
    @staticmethod
    def configured_targets_or_default(targets: Set[str]) -> Set[str]:
        return targets or {DEFAULT_VERSION}
```

Reply:
Is this method really needed? Using `*(self.configured_{build,tests}_targets or {DEFAULT_VERSION})` bellow would result in less code.

Reply:
I'm going to look into it… Probably going to prepare some PRs to `hello-world` for Friday, to put it on stage and test it out, cause with the tests the targets are non-trivial to think about :D

Or I could just fetch the Copr targets and do not care about the fact that Copr can do it by itself :thinking: (sounds easier tbf)

Reply:
Sorry, I don't get it :) This method takes an argument and returns it if it's true-ish or returns a default otherwise. My question was whether this simple expression really needs to be wrapped in a method, instead of just using it straight away where it's needed.